src/commands/contextMenu/fxtwitter.ts

```diff
@@ -1,5 +1,4 @@
 import { ApplicationCommandType, ContextMenuCommandBuilder } from 'discord.js';
-import { URL } from 'node:url';
```

Comment:
I think this was technically unnecessary - just needed to update `@types/node` for the latest ts version.

Reply:
Oh neat!

Reply:
Reverted. Except now it looks like discord.js doesn't like the new typescript version :skull:

Reply:
Updating that first might do the trick tho, idk